libraries/aws_s3_bucket.rb

```diff
@@ -23,6 +23,9 @@ def initialize(opts = {})
     catch_aws_errors do
       begin
         @region = @aws.storage_client.get_bucket_location(bucket: @bucket_name).location_constraint
+        # LocationConstraint "EU" correlates to the region "eu-west-1", but region "EU" does not exist as a "region", only a LocationConstraint
+        # this currently is the only Location constraint that can have either of 2 values "EU" or "eu-west-1".  But only "eu-west-1" is a region
+        @region = 'eu-west-1' if @region == 'EU'
```

Comment:
Okay makes sense .. and it looks like this is the only non-region based value that is being returned currently from location_constraint https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/S3/Client.html#get_bucket_location-instance_method